### PR TITLE
Feature/5130 remove react auth shared

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # BLUI-5130

#### Changes proposed in this Pull Request:

- Removed dependency on shared auth
- Removed references of  shared auth
- Updated scripts

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

- Tests will fail, because components are using react auth shared functionalities

### CODE REVIEW

File: login-workflow/package.json [hunk 1]
```diff
@@ -32,12 +32,10 @@
...
-        "initialize": "bash scripts/initializeSubmodule.sh",
         "install:dependencies": "yarn install:dependencies-ios",
-        "install:dependencies-ios": "yarn && cd example && yarn && cd ios && pod install && cd ../../shared-auth && yarn && cd .. && yarn link:shared",
-        "install:dependencies-android": "yarn && cd example && yarn && cd ../shared-auth && yarn && cd .. && yarn link:shared",
+        "install:dependencies-ios": "yarn && cd example && yarn && cd ios && pod install",
+        "install:dependencies-android": "yarn && cd example && yarn",
         "link:workflows": "bash ./scripts/linkWorkflow.sh",
-        "link:shared": "bash ./scripts/linkShared.sh",
         "start:example": "yarn start:example-ios",
         "start:example-ios": "yarn initialize && yarn install:dependencies-ios && yarn build && yarn link:workflows && cd example && yarn ios",
```
Comment (JeffGreiner-eaton) [inline login-workflow/package.json:40]: you can remove yarn initialize for both iOS and android

**Comment by JeffGreiner-eaton** [comment]
for me I have issue with install ios, not part of this PR so can't verify running

image

and for whatever reason android failed to start daemon but only happens in this repo not others.

**Comment by JeffGreiner-eaton** [comment]
will you be porting over the code from shared auth?

-Duplicate the code that currently exists in the react-auth-shared repository to be local inside of the react native workflows package

**Comment by ektaghag-eaton** [comment]
Hi @surajeaton, please take latest changes from release/R31.

**Comment by surajeaton** [comment]
> > Hi @surajeaton, please take latest changes from release/R31.
> 
> plus update start scripts to exclude initialize, or are we planning something else?

Removed initialize from the start script

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good to me as starting point

**Pull request merged**